crates/rspack_error/src/graphical.rs

```diff
@@ -0,0 +1,1007 @@
+#![allow(dead_code)]
+#![allow(clippy::unwrap_used)]
+#![allow(clippy::needless_lifetimes)]
+
+/// THIS FILE IS ORIGINALLY FROM THE MIETTE PROJECT:
```

Comment:
Is there a way not to fork this?

Reply:
It's not as customizable as I thought it would be.